examples/plot_bids_analysis.py

```diff
@@ -0,0 +1,114 @@
+
+#############################################################################
+# Obtain automatically FirstLevelModel objects and fit arguments
+# --------------------------------------------------------------
+# From the dataset directory we obtain automatically FirstLevelModel objects
+# with their subject_id filled from the BIDS dataset. Moreover we obtain
+# for each model a dictionary with run_imgs, events and confounder regressors
+# since in this case a confounds.tsv file is available in the BIDS dataset.
+# To get the first level models we only have to specify the dataset directory
+# and the task_id as specified in the file names.
+task_id = 'languagelocalizer'
+models, fit_kwargs = first_level_models_from_bids(
+    data_dir, task_id, preproc_space='MNI152nonlin2009aAsym',
+    preproc_variant='smoothResamp')
+
+#############################################################################
+# Quick sanity check on fit arguments
+# -----------------------------------
+# Normally you might want to do some additional processing on event files
+# or you might want to create or extend confound regressors. In this example
+# dataset none of that is necessary, so we will just go through the
+# experimental paradigm and check all is fine for the first subject.
+
+############################################################################
+# We just expect one run img per subject.
+print([os.path.basename(run) for run in fit_kwargs[0]['run_imgs']])
+
+##############################################################################
+# The only confounds stored are regressors obtained from motion correction. As
+# we can verify from the column headers of the confounds table corresponding
+# to the only run_img present
+print(fit_kwargs[0]['confounds'][0].columns)
+
+############################################################################
+# During this acquisition the subject read blocks of sentences and
+# consonant strings. So these are our only two conditions in events.
+# We verify there are n blocks for each condition.
+print(fit_kwargs[0]['events'][0]['trial_type'].value_counts())
+
+############################################################################
+# First level model estimation
+# ----------------------------
+# Now we simply fit each first level model and plot for each subject the
+# contrast that reveals the language network (language - string).
+fig, axes = plt.subplots(nrows=2, ncols=5)
+for midx, (model, model_kwargs) in enumerate(zip(models, fit_kwargs)):
+    model.fit(**model_kwargs)
+    zmap = model.compute_contrast('language-string')
+    plotting.plot_glass_brain(zmap, colorbar=False, threshold=norm.isf(0.001),
+                              title=('sub-' + model.subject_label),
+                              axes=axes[midx / 5, midx % 5],
+                              plot_abs=False, display_mode='x')
+fig.suptitle('subjects z_map language netowrk (language - string)')
+plt.show()
+
```

Comment:
Definitely for the group results map, but adding the colorbar for each small brain looked visually unappealing when I tried. Anyway I show all the brains just for illustrative purposes to compare visually the patterns with the group result. An option would be to make a common colorbar with the max stat of all images, but seems to me to complicate the code in the example without adding much value to it.